packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware.ts

```diff
@@ -181,6 +182,20 @@ const simpleBuyOriginDictionary = (rawOrigin: SBShowModalOriginType) => {
   }
 }
 
+const simpleBuyPaymentTypeDictionary = (rawPaymentType: SBPaymentTypes) => {
+  switch (rawPaymentType) {
+    case 'USER_CARD': {
+      return 'PAYMENT_CARD'
+    }
+    case 'LINK_BANK': {
+      return 'BANK_TRANSFER'
+    }
+    default: {
+      return rawPaymentType
+    }
+  }
+}
```

Comment:
Nice, btw I think we might consider to do a big cleanup with this would be cool to add tech debt ticket for it ;)

Reply:
Hello how's it going